Makefile.core.mk

```diff
@@ -483,9 +483,10 @@ gitleaks: $(GITLEAKS) ## Download gitleaks to bin directory.
 $(GITLEAKS): $(LOCALBIN)
 	@test -s $(LOCALBIN)/gitleaks || GOBIN=$(LOCALBIN) go install github.com/zricethezav/gitleaks/v8@${GITLEAKS_VERSION}
 
+HELM_PLATFORM ?= openshift
```

Comment:
Could you please add a comment with more context and valid options?

Reply:
I Improved to make the set of the variable and for what is used more clear

Reply:
Ok, I changed the variable name to OPENSHIFT_PLATFORM with expected values true or false, if set to true with set the flag if not it will not add the flag

Reply:
Like this it will be more easy to understand and make more sense according what we are setting or not setting in the helm command